standalone/index.js

```diff
@@ -0,0 +1,31 @@
+const getPort = require("get-port");
+const argv = require("yargs").argv;
+const fs = require('fs');
+
+const AietesServer = require("../mock-server");
+
+standalone = async () => {
+    console.info("Starting Aietes standalone");
+
+    const port = argv.port || await getPort();
+
+    const responseFileName = argv.json;
+    let jsonResponse;
+    if (responseFileName) {
+        jsonResponse = JSON.parse(await fs.readFile(responseFileName, 'utf8'));
```

Comment:
Should we implement something if JSON.parse throws an error because the Json inside the file is not valid?

Reply:
That does sound like a good idea. Though I think the only thing we can do is output a (hopefully) helpful error and exit gracefully.

Reply:
Here or next PR?

Reply:
Come to think of it: what do you mean? Just catch the error that `readFile` may throw or validate the structure of the JSON read in before passing it to Aietes?
We could do both.

Reply:
I'd suggest doing the validation in a different PR though.

Reply:
Deal.